projects/vendor/src/yotpo/yotporeview/yotporeview.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, ElementRef } from '@angular/core';
+import { BaseyotpoComponent } from './../baseyotpo/baseyotpo.component';
+import { ProductService, RoutingService } from '@spartacus/core';
+
+@Component({
+  selector: 'cx-yotporeview',
+  templateUrl: './yotporeview.component.html',
+  styleUrls: ['./yotporeview.component.css']
+})
+export class YotporeviewComponent extends BaseyotpoComponent {
```

Comment:
Remove the inheritance (see parent class comment)

Reply:
fixed